plugins/qtosgrave/osgviewerwidget.cpp

```diff
@@ -1155,7 +1197,8 @@ void QOSGViewerWidget::_PanCameraTowardsDirection(double delta, const osg::Vec3d
 
 void QOSGViewerWidget::MoveCameraZoom(float factor, bool isPan, float panDelta)
 {
-    if(isPan) {
+    std::cout << "MOVE CAMERA ZOOM factor = " << factor << " isPan = " << isPan << " panDelta = "  << panDelta << "is ortho = " << IsInOrthoMode() << std::endl;
```

Comment:
Use `RAVELOG`, probably verbose

Reply:
I actually removed those message now, they were debug and I forgot to remove.